### ISSUE TITLE
changes.txt is empty

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/hookes-law#8]
changes.txt is empty in Hook's Law, is this OK?  

The PhET Development Overview states that this file is visible to all PhET users after deployment, so I wanted to make sure.

**Comment by pixelzoom** [comment]
@jessegreenberg changes.txt has a complicated history. Got a minute? :)

changes.txt was used in Java/Flash sims by some developers (mostly me) as a way to record the changes that they made during development.  This was done mostly because Unfuddle SVN made it difficult to see commit messages. Most of the entries in changes.txt were in developer-speak and not for public consumption. 

At some point PhET decided that it would be nice include change entries that were visible to the end-user, for example to indicate bug fixes or new features.  So we began placing a '>' at the beginning of any line that was intended to be visible by the end-user, and those lines were extracted and displayed on the website when you click a "What's New" link.   For example:

When we moved to HTML5 development, changes.txt came along (again used mostly by me.) At some point we decided that GitHub provides a nice way to see commit comments, so changes.txt was dropped (even by me :) for developer-speak comments.  I believe that it continues to live on for the possibility of including end-user comments - though I don't believe that functionality has been implemented by the website yet. 

This sim has an empty changes.txt because it's included in the simula-rasa template, and because (since the sim hasn't been released) there are no fixed or enhancements to inform the end-user about.

Let me know if you have an questions.

Closing.
